src/main/java/apoc/util/s3/S3Aws.java

```diff
@@ -46,4 +67,20 @@ public long getLength() {
             }
         };
     }
+
+    private static AWSCredentialsProvider getCredentialsProvider(
+            final String accessKey, final String secretKey, final String sessionToken) {
+
+        if (Objects.nonNull(accessKey) && !accessKey.isEmpty()
```

Comment:
```suggestion
        if (StringUtils.isNotBlank(accessKey)
```

and the same as for others?